src/conf.py

```diff
@@ -54,9 +56,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.0.1a1'
+version = u'1.0'
```

Comment:
May as well change this to 2019.01, since that is the last big tag we had.

Reply:
Done!